### ISSUE TITLE
Mirror Window not launching with S9+

### ISSUE THREAD

**Issue by SpitFire1992** [Genymobile/scrcpy#1665]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 10
 - scrcpy version: 1.16
 - installation method: Windows release
 - device model: Samsung Galaxy S9+
 - Android version: 10

**Describe the bug**
It worked fine a few days ago, i recently installed a new update on my phone and since then, i can launch the the terminal version of scrcpy and it recognizes the phone but wont launch the second window that actually mirrors the screen.

thats it.

```
`INFO: scrcpy 1.16 <https://github.com/Genymobile/scrcpy>
C:\adb\scrcpy-server: 1 file pushed, 0 skipped. 30.6 MB/s (33622 bytes in 0.001s)
[server] INFO: Device: samsung SM-G965F (Android 10)`
```

**Comment by rom1v** [comment]
An encoder problem on the tablet probably.

Does it work with `scrcpy -m 800`?

Does `screenrecord` work:

```
adb shell screenrecord /sdcard/file.mp4
adb pull /sdcard/file.mp4
```
?

**Comment by SpitFire1992** [comment]
Hey, thank you.
Screenrecord works. 
Your first command isn't working.

**Comment by rom1v** [comment]
Hmm... Does some older version of scrcpy work? (for example v1.5, v1.9, v1.13)

**Comment by SpitFire1992** [comment]
Output of 1.5
`* daemon not running; starting now at tcp:5037
* daemon started successfully
scrcpy-server.jar: 1 file pushed.`
and then nothing happens, waited a good minute.

Output of 1.9
`* daemon not running; starting now at tcp:5037
* daemon started successfully
C:\adb\scrcpy-server.jar: 1 file pushed. 4.0 MB/s (22470 bytes in 0.005s)
INFO: Initial texture: 1080x2216`
and then, nothing happens, also waited for 1+ minute. after 2 minutes or so it actually starts working. were the older releases slower? Because with the newer release it basically opened the mirrorwindow instantly. Got a fresh windows installation so i should have clustered it up yet.

1.13
`INFO: scrcpy 1.13 <https://github.com/Genymobile/scrcpy>
* daemon not running; starting now at tcp:5037
* daemon started successfully
C:\adb\scrcpy-server: 1 file pushed. 5.1 MB/s (27694 bytes in 0.005s)
[server] INFO: Device: samsung SM-G965F (Android 10)
`
Basically what happens with the latest release.

**Comment by rom1v** [comment]
Wow, that's weird.

So we have to find were it is blocked on the device.

Are you comfortable with adding logs and [building](https://github.com/Genymobile/scrcpy/blob/master/BUILD.md) the app/server?

As a first step, you could just execute `scrcpy -Vdebug` (but I think it will not help a lot).

(when it is "blocked", what happens if you press <kbd>Enter</kbd> in the terminal? I already had think kind of issues with Windows terminal in general)

**Comment by SpitFire1992** [comment]
You mean the catlog? Checked that out before and it added literally nothing (wich is weird since it should add that it started scrcpy, no? Never build the app/server myself, i check it out tho, if its not too complicated i give it a try.

The command  `scrcpy -Vdebug` prints out:
```
INFO: scrcpy 1.16 <https://github.com/Genymobile/scrcpy>
DEBUG: Using server (portable): C:\adb\scrcpy-server
C:\adb\scrcpy-server: 1 file pushed, 0 skipped. 33.1 MB/s (33622 bytes in 0.001s)
DEBUG: Screensaver enabled
[server] INFO: Device: samsung SM-G965F (Android 10)
DEBUG: Starting stream thread
DEBUG: Starting controller thread
```
Is it supposed to do more? I keep the console open for now.

It just jumps to the next line when i press Enter, nothing else.

**Comment by rom1v** [comment]
> You mean the catlog?

(logcat)

> Checked that out before and it added literally nothing (wich is weird since it should add that it started scrcpy)

Oh, so the output of `adb logcat` is stuck, whatever you do on your phone?
This kind of problem (independant of scrcpy) has already been observed (I can't find the issue anymore), and it "prevented" scrcpy to work.

> i check it out tho, if its not too complicated i give it a try.

OK, let me know :+1:

The goal would be to find on which Java line the server is blocked.

**Comment by SpitFire1992** [comment]
Hey, seems to be working normally today. Apparently multiple windows machines had problems yesterday, from machine being super slow to drivers and programs not starting at all, after installing a little update today everything, including scrcpy, seems to be working normally again. 
Thanks for your help!
